glyphs-reader/src/font.rs

```diff
@@ -5,6 +5,7 @@
 //! where it gets serialized to more Rust-native structures, proc macros, etc.
 
 use std::collections::{BTreeMap, HashSet};
+use std::fmt::Debug;
```

Comment:
is this required? isn't Debug already in the preamble?

Reply:
Err. Yes. I wonder how I managed to do that!